data_architect_misc/data_transformer/constants/comp_harm_constants.py

```diff
@@ -147,23 +147,32 @@
 }
 
 CATEGORY_MAPPINGS = {
+    "(?i).*air.*care.*": "Home Care",
+    "(?i).*Bath.*": "Home Care",
     "(?i)^HC$": "Home Care",
     "(?i)HOME.*": "Home Care",
     "(?i).*HOME.*CARE.*": "Home Care",
     "(?i).*HOUSEHOLD.*": "Home Care",
     "(?i).*Laundry.*": "Home Care",
+    "(?i).*Toiletries.*": "Home Care",
+    "(?i).*Cleaning.*agent.*": "Home Care",
 
     "(?i)^OC$": "Oral Care",
     "(?i).*ORAL.*": "Oral Care",
     "(?i).*ORAL.*CARE.*": "Oral Care",
+    "(?i).*Dental.*": "Oral Care",
 
     "(?i).*BEAUTY.*": "Personal Care",
+    "(?i).*BODY.*CARE.*": "Personal Care",
     "(?i).*Cosmetics.*": "Personal Care",
+    "(?i).*Deodorant.*": "Personal Care",
+    "(?i).*FACE.*CARE.*": "Personal Care",
+    "(?i).*HAIR.*CARE.*": "Personal Care",
+    "(?i).*Hair.*": "Personal Care",
```

Comment:
Could match to chair. or worse, to 'hair declogger' and that is home care product. So don't do this in standard mapping. Maybe you can add '\shair\s' : 'personcal care' in Malaysia, but even that is a bit risky.